src/lucky_migrator/create_table_statement.cr

```diff
@@ -144,4 +167,8 @@ class LuckyMigrator::CreateTableStatement
       " NOT NULL"
     end
   end
+
+  def references(table_name)
+    table_name == "" ? "" : " REFERENCES #{table_name}"
```

Comment:
I think regular ifs are a bit easier to read. Could we do something like this instead:

```crystal
if table_name
  " REFERENCES #{table_name}"
else
  ""
end
```

Reply:
I see what you mean, this is easier to scan